jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/handler/ResourceHandlerTest.java

```diff
@@ -2798,8 +2821,12 @@ public void testListingXSS() throws Exception
 
         /*
          * Intentionally bad request URI. Sending a non-encoded URI with typically
-         * encoded characters '<', '>', and '"'.
+         * encoded characters '<', '>', and '"', using the path-param feature of the
+         * URI spec to still produce a listing.  This path-param value should not make it
+         * down to the ResourceListing.getAsXHTML() method.
          */
+        // TODO: investigate why ResourceHandler isn't producing a directory listing when asked here.
```

Comment:
Please open an issue about this problem.

Reply:
This is now a bogus TODO.
This was actually fixed in other PRs already.
Removing both TODOs here.